### DISCUSSION TITLE
2.0.3-表单焦点问题

### DISCUSSION THREAD

**Discussion by ReganHe** [element-plus/element-plus#6309] [Q&A]
升级到2.0.3，使用表单组件时，有两个问题（焦点相关）：

1. 打开表单时，默认聚焦于第一个控件
2. 在表单中，点击日期区间控件，弹出选择日期区间浮层，无法正常关闭

回滚到2.0.2后，问题消失，可正常使用

**Comment by msidolphin** [comment]
关注下 #6294，已经在处理了
